src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final MoveStrategy moveStrategy = new RacingMoveStrategy();
+    private final String name;
```

Comment:
```suggestion
    private static final MoveStrategy moveStrategy = new RacingMoveStrategy();
    
    private final String name;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가하면 가독성이 상승합니다!